Add some details to wiz docu (#21628)
home-assistant/home-assistant.github.io: source/_integrations/wiz.markdown

---

The WiZ integration allows you to control your WiZ lights and smart sockets.
The devices are set up through your Wi-Fi network and don't need any additional bridge or gateway.

{% include integrations/config_flow.md %}

## Connect WiZ devices to your network

To connect a WiZ device to your Wi-Fi network, please follow the instructions in the [WiZ app](https://www.wizconnected.com/en/consumer/app/) (available for iOS and Android).

> If you have further questions, please have a look into the [WiZ Support Page](https://www.wizconnected.com/en/consumer/support/).

Reviewer: ```suggestion
To connect a WiZ device to your Wi-Fi network, please follow the instructions in the [WiZ app](https://www.wizconnected.com/en/consumer/app/) (available for iOS and Android).
If you have further questions, please have a look at the [WiZ Support Page](https://www.wizconnected.com/en/consumer/support/).
```

### Enable local connectivity

> The integration needs to communicated locally with the WiZ device. This setting **Allow local communication** can be disabled or enabled in WiZ app.

Reviewer: ```suggestion
The integration needs to communicate locally with the WiZ device. This setting **Allow local communication** can be disabled or enabled in WiZ app.
```

This setting should be enabled by default.

Steps to enable:

1. Open the WiZ app on your phone
2. Click on the settings menu in the upper-left corner
3. Scroll down to the security settings

> 4. Enable the switch **Allow local communication**

Reviewer: Should this be listed as prerequisites above the config flow instructions?

Author: The connection part should be above. Because this step is necessary if you plug in a device the first time.

> 4. Enable the switch **Allow local communication**

Reviewer: ```suggestion
1. Open the WiZ app on your phone.
2. Click on the settings menu in the upper-left corner.
3. Scroll down to the security settings.
4. Enable the switch **Allow local communication**.
```

> 
> ## Using discovery in container
> 
> For discovering WiZ devices in your network, the Home Assistant container must be able to send and receive UDP broadcast packages.
> This requires that the container has access to the host network. This can be achieved by setting `--network host` as start parameter for the Home Assistant Docker container.
> More details can be found in the [official Docker documentation](https://docs.docker.com/network/host/).

Reviewer: This section can be removed, we do not support running it without host networking.

```suggestion
```
